apps/member-profile/app/shared/components/profile.personal.tsx

```diff
@@ -5,6 +5,23 @@ import { Country } from '../core.ui';
 import { CityCombobox, CityComboboxProps } from './city-combobox';
 import { EthnicityMultiCombobox } from './ethnicity-combobox';
 
+export function AllowEmailShareField({
```

Comment:
I put this function in this file because it imitates the style of the other checkboxes in this file, but I also wasn't sure because technically this checkbox is implemented in `_profile.profile.emails.tsx` not `_profile.profile.personal.tsx`. There isn't a file `profile.emails.tsx` yet, would this be worth creating just for this function?

Reply:
Honestly, I'm cool not abstracting this component into a separate file. I'm not completely sold on that pattern yet, aside from reusable components.

Feel free to just include this directly in the form, ie:

```tsx
<RemixForm>
  <Form.Field ...>
</RemixForm>
```